x-pack/plugins/security_solution/public/resolver/view/panels/event_detail.tsx

```diff
@@ -255,6 +263,15 @@ function EventDetailBreadcrumbs({
         text: <DescriptiveName event={event} />,
       },
     ];
+
+    if (nodeName) {
```

Comment:
❔ Do we want to put in a placeholderish crumb even if we can't squeeze a name out of the node? Like a `...` or something you can click to to get to the process detail panel?

Reply:
I like that idea. I'll put it in a follow up PR to avoid waiting for CI again and holding this up. Thanks!